sky/utils/cli_utils/status_utils.py

```diff
@@ -80,6 +80,7 @@ def show_status_table(cluster_records: List[_ClusterRecord],
         StatusColumn('ZONE', _get_zone, show_by_default=False),
         StatusColumn('STATUS', _get_status_colored),
         StatusColumn('AUTOSTOP', _get_autostop),
+        StatusColumn('HEAD_IP', _get_head_ip),
```

Comment:
It seems showing in the `sky status` as well.

Reply:
fixed